packages/client/src/components/form/FormFieldGenerator.tsx

```diff
@@ -131,10 +131,7 @@ const FormItem = styled.div<{
 }>`
   animation: ${fadeIn} 500ms;
```

Comment:
Delete? The form fields don't and shouldn't animate in

Reply:
@jpye-finch it looks like they do and you can see it happening in the birth and death forms in QA.  My guess is that back in the day when user testing in Bangladesh, that the form sections rendered so quickly that users did not notice the section change.  This very fast fade-in helps the user understand that there is a new page if they are just clicking through the form without entering data.  I think we should leave it in here

Reply:
oh interesting. ok cool